modules/search/instant-search/components/notice.scss

```
@@ -0,0 +1,18 @@
+@import 'modules/calypsoify/_studio-wpcom.scss';
```

Instead of importing from another module, maybe we should try to import directly from `@automattic/color-studio` now that you've added it to the dependencies?

```suggestion
@import '~@automattic/color-studio/dist/color-variables.scss';
```